doc/ref_arch/kubernetes/chapters/chapter04.md

```diff
@@ -152,7 +180,8 @@ To support the isolation of the resources used by the infrastructure from the re
 <a name="4.5"></a>
 ## 4.5 Networking solutions
```

Comment:
any requirements regarding a proper network blueprint? separate VLANs for infra, proper bonding, physical separation of infra and app traffic etc.?

Reply:
The requirements are in chapter02.md - what we need to add into this table is the configuration specification for the Kubernetes networking solution.  So yes to what you've said - but we need to define the configuration spec in this chapter, based on requirements defined in chapter02.md